### PR TITLE
Label cut off

### PR DESCRIPTION
**Pull request by freshking**
## Description

Allow font size to adjust to given space

## Link to Jira

https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4770

#### Before:

IMG_0018

#### After:

IMG_0017

### CODE REVIEW

**Review by tcfos — APPROVED** [review]

File: src/xcode/ENA/ENA.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -5595,7 +5595,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				MARKETING_VERSION = 1.12.0;
+				MARKETING_VERSION = 1.13.0;
```
Comment (whiskey) [inline src/xcode/ENA/ENA.xcodeproj/project.pbxproj:5598]: I always thought the CI or someone later in the release process sets this label… 🤷‍♂️

**Review by whiskey — APPROVED** [review]

**Comment by whiskey** [comment]
One thing. If this PR goes into the 1.13.x release, please update the milestone here!

File: src/xcode/ENA/ENA.xcodeproj/project.pbxproj (hunk 1 above)
Comment (freshking) [reply src/xcode/ENA/ENA.xcodeproj/project.pbxproj:5598]: I have no idea. I accidentally create this PR for the 1.13 branch, then switched to 1.12. That switch added these changes.

Comment (marcussc) [reply src/xcode/ENA/ENA.xcodeproj/project.pbxproj:5598]: Yeah if you merged 1.13 into your branch the version change to 1.13 is in your PR as well. Please revert your changes to the *.pbxproj

Comment (freshking) [reply src/xcode/ENA/ENA.xcodeproj/project.pbxproj:5598]: Will do. Thanks.

**Pull request merged**